cli/src/main/scala/tfr/Cli.scala

```diff
@@ -65,7 +68,11 @@ object Cli extends CaseApp[Options] {
         // we should display errors somehow
         case Right(example) => example
       }
-      .showLines(Console.out)
+      .showLines(Console.out)(
```

Comment:
given our new use case here, i think it will be better if we de-sugar this into smth like:
```scala
.map { example =>
 if ... else ...
}
.lines(Console.out)
```